src/main/java/ConnectionManagement/Connection.java

```diff
@@ -0,0 +1,14 @@
+package ConnectionManagement;
+
+/**
+ * Created by alfatihmukhtar on 2/3/17.
+ */
+public interface Connection {
```

Comment:
Connection should extend the Closeable interface. This way all implementations of Connection are guaranteed to be closeable.